dali/pipeline/pipeline.cc

```diff
@@ -198,6 +198,8 @@ int Pipeline::AddOperator(OpSpec spec, const std::string& inst_name, int logical
     next_logical_id_ = logical_id + 1;
   }
 
+  // we modify spec so strip const from it
+  OpSpec &spec = const_cast<OpSpec &>(const_spec);
```

Comment:
Wait, what? This is breaking any sane contract.
What if I did:
```
OpSpec some_spec = ...;

Pipeline x(...), y(...);
x.AddOperator(some_spec);
// optionally modify some_spec
y.AddOperator(some_spec);
```

Reply:
done